Authentication/SignUp/View/SignupControllerDelegate.swift

```diff
@@ -41,7 +41,7 @@ extension SignupControllerDelegate {
     public var shouldDisplaySignupErrorWithAlert: Bool { return true }
     
     public func signupControllerWillExecuteSignUp(controller: SignupController) {
-        if let errorLabel = controller.signupView.signupErrorLabel {
+        if let errorLabel = controller.signupView.signUpErrorLabel {
             errorLabel.text = " "
```

Comment:
Ah, sorry. I changed it so to be easier to associate with the sign up button which does the action, since the label is for that action's error, not the whole signup page.
Do I change it back?